pkgs/top-level/all-packages.nix

```diff
@@ -22169,7 +22169,7 @@ with pkgs;
   ;
   mariadb = mariadb_106;
 
-  mongodb = hiPrio mongodb-3_4;
+  mongodb = hiPrio mongodb-5_0;
```

Comment:
I think it's a good idea to document the default version change in a changelog entry.

Reply:
Will do.